webull/webull.py

```diff
@@ -248,8 +250,22 @@ def refresh_login(self):
             self._access_token = result['accessToken']
             self._refresh_token = result['refreshToken']
             self._token_expire = result['tokenExpireTime']
+            if save_token:
+                self._save_token(result)
         return result
 
+    def _save_token(self, token=None, path=None):
```

Comment:
method signature allows for specifying a path but this is a protected method and the methods that call it don't allow the user to specify a path so this option would never get used. Maybe allow the `login` method accept an optional `save_token_path` which can then be passed along?